src/yargs-options.js

```diff
@@ -47,6 +47,11 @@ const options = {
     type: 'boolean',
     default: false,
   },
+  'mv3-background-service-worker': {
```

Comment:
```suggestion
  'enable-background-service-worker': {
```

or something like that maybe?